LabExT/Movement/MoverNew.py

```diff
@@ -358,6 +360,51 @@ def z_lift(self, lift: float) -> None:
     #   Movement Methods
     #
 
+    @assert_connected_stages
+    def move_absolute(
+        self,
+        movement_commands: Dict[Orientation, Type[ChipCoordinate]],
+        chip: Type[Chip],
+        wait_for_stopping: bool = True
+    ) -> None:
+        """
+        Moves the stages absolutely in the chip coordinate system.
+
+        A collision-free trajectory is calculated.
+
+        Parameters
+        ----------
+        movement_commands : Dict[Orientation, Type[ChipCoordinate]]
+            A mapping between orientation and target chip coordinate.
+            For example, if the mapping `Orientation.LEFT: ChipCoordinate(1,2,3)` exists, the left stage is moved to the chip co-ordinate x=1, y=2, z=3
+        wait_for_stopping: bool = True
+            Whether each stage should have completed its movement before the next one moves.
+
+        Raises
+        ------
+        MoverError
+            If an orientation has been given a target, but no stage exists for this orientation.
+        LocalMinimumError
+            If the path-finding algorithm makes no progress
+             i.e. does not converge to the target coordinate.
+        """
+        path_planning = PathPlanning(chip)
+
+        # Set up Path Planning
+        for orientation, target in movement_commands.items():
+            calibration = self._get_calibration(orientation=orientation)
+            if calibration is None:
+                raise MoverError(
+                    f"No {orientation} stage configured, but target coordinate for {orientation} passed.")
+
+            path_planning.set_stage_target(orientation, target)
+
+        # Move stages on safe trajectory
+        for calibration_waypoints in path_planning.trajectory():
+            for calibration, waypoint in calibration_waypoints.items():
+                with calibration.perform_in_chip_coordinates():
+                    calibration.move_absolute(waypoint, wait_for_stopping)
```

Comment:
Suggestion: all stages/calibrations should move at the same time to their next waypoint (i.e. call `calibration.move_absolute` with `wait_for_stopping=False`. But then wait for all movements completed (incl. SW limited timeout!) before assigning all calibrations a new waypoint.

Reply:
Added!